module#2/src/api/middlewares/user-middleware.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response, NextFunction } from "express";
+import { isUserAlreadyExist } from "../../utils/user-operations";
+import { ERROR_MESSAGE, HTTP_STATUSES } from "../../constants";
+
+export const userValidation = (req: Request, res: Response, next: NextFunction) => {
+  const isUserExist = isUserAlreadyExist(req.body.login);
```

Comment:
Sorry, didn't see this. Why can't we use isUserAlreadyExist() directly in if-statement? Why do you create separate constant?

Reply:
I don't know. I thought it will be better, more readable, but also use directly in condition good variant. Done) Thanks)))